doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -50,9 +61,156 @@ EVP_PKEY_fromdata() return 1 for success and 0 or a negative value for
 failure.  In particular a return value of -2 indicates the operation is
 not supported by the public key algorithm.
 
+=head1 EXAMPLES
+
+These examples are very terse for the sake of staying on topic, which
+are the EVP_PKEY_fromdata() set of functions.  In real applications,
+BIGNUMs would be handled, and converted to byte arrays with
+BN_bn2nativepad(), but that's off topic here.
+
+=begin comment
+
+TODO Write a set of cookbook documents and link to them.
+
+=end comment
+
+=head2 Creating an RSA keypair using raw key data
+
+ #include <openssl/evp.h>
+
+ /*
+  * These are extremely small to make this example simple.  A real
+  * and secure application will not use such small numbers.  A real
+  * and secure application is expected to use BIGNUMs, and to build
+  * this array dynamically.
+  */
+ const unsigned long rsa_n = 0xbc747fc5;
+ const unsigned long rsa_e = 0x10001;
+ const unsigned long rsa_d = 0x7b133399;
+ const OSSL_PARAM[] = {
+     OSSL_PARAM_ulong("n", &rsa_n),
+     OSSL_PARAM_ulong("e", &rsa_e),
+     OSSL_PARAM_ulong("d", &rsa_d),
+     OSSL_PARAM_END
+ };
+     
+ int main()
+ {
+     EVP_PKEY_CTX *ctx = EVP_PKEY_CTX_new_from_name(NULL, "RSA", NULL);
+     EVP_PKEY *pkey = NULL;
+
+     if (ctx == NULL
+         || !EVP_PKEY_key_fromdata_init(ctx)
+         || !EVP_PKEY_fromdata(ctx, &pkey, params))
+         exit(1);
```

Comment:
by indenting the || above the exit is more visual, that was Andy's reasoning for the style.
Would you mind adding extra indents here, @levitte ?

Reply:
It wasn't Andy's idea. And yes I do mind, as it means I must either battle emacs' cc-mode indentation, or turn it off and then waste time indenting manually.

Reply:
Or oh wait...  in manual, yeahok, I have to do it manually anyway.

Mind you, though, that I find that style jarring, and also that even those who do this don't do it consistently, i.e. they do after `if` but not after `else if`

Reply:
> And yes I do mind, as it means I must either battle emacs' cc-mode indentation, or turn it off and then waste time indenting manually.

Same holds for me. Also, I think that the `|| ` prefix already adds sufficient extra indentation to the followup conditions.

Reply:
> it means I must either battle emacs' cc-mode indentation, or turn it off and then waste time indenting manually.

I am afraid you are gonna lose this battle...

Reply:
I think @DDvO pointed aout an Emacs solution for this problem, though.

Reply:
Reference please.  I haven't seen that solution.  I have said a few times that if someone comes up with a viable emacs solution, I will consider it.

Reply:
I looked around in other manuals where we do have examples with multiline `if` conditions, and we don't have extra indents in any of them.

We hopefully write manuals _for others_, more than for ourselves, so I think it's wiser to use a style that's commonly reachable rather than one that requires some extra hackery to follow, be it in code or in mind.  So in the end, I'm not keen in making the change proposed here.

Reply:
@levitte have you considered https://github.com/openssl/openssl/issues/10725#issuecomment-585675825
does that resolve your concerns?

Anyway this unresolved conflict starts to poison the whole openssl project.
I do not really understand why this fight is necessay, without the
constant symmetry breaks in the coding style we could concentrate
to other matters, like the issue I raised on secrity@openssl.org two
days ago, and nobody took it serious... yet I believe that is a major
security issue.